server/container/guice/memory-guice/pom.xml

```diff
@@ -106,6 +106,11 @@
             <artifactId>james-server-guice-webadmin-mailbox</artifactId>
         </dependency>
         <dependency>
+            <groupId>${project.groupId}</groupId>
```

Comment:
Why? Where do you use it?

Reply:
You just need this dependency were you use it:

JMAP integration tests + cassandra and memory as well as Mailet integration tests, if I follow correctly.